src/os/host-start.r

```diff
@@ -423,6 +421,8 @@ host-start: function [
         ]
     ]
 
+    boot-embedded: get-encap system/options/boot
+
```

Comment:
Uh, right, and that could be handled by another commit. So this should be good by itself.